tests/main/snap-repair/uc16.json

```diff
@@ -0,0 +1,6 @@
+{
+    "revision": "1",
```

Comment:
the revision 1 are strange,   we start at 0, unless you have an actual test that does retry revising the assertion in between